src/containers/Folder.jsx

```diff
@@ -24,14 +37,37 @@ class Folder extends Component {
 
   render (props, state) {
     if (props.isFetching === true) {
-      return <p>Loading</p>
+      return (
+        <div role='contentinfo'>
+          <p>Loading</p>
+        </div>
+      )
     }
-    return <FileList {...props} {...state} />
+    const isTrashContext = props.context === TRASH_CONTEXT
```

Comment:
For clarity, I would suggest to also have a `isFilesContext` variable here. It makes things more readable and paves the way for future additional screens.